packages/web/src/components/gcds-footer/gcds-footer.stories.tsx

```diff
@@ -46,27 +46,27 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-footer
-  display="${args.display}"
-  contextual-heading="${args.contextualHeading}"
-  contextual-links='${args.contextualLinks}'
-  sub-links='${args.subLinks}'
-  lang="${args.lang}"
+  ${args.display != "compact" ? `display="${args.display}"` : null}
```

Comment:
Similar to the button type, the footer display is required but it doesn't show up by default. Since it has a default value, does this prop still need to be required?